spec/controllers/bookings_controller_spec.rb

```diff
@@ -7,4 +7,26 @@
       expect(response).to render_template("index")
     end
   end
+
+  describe 'POST bookings#create' do
+    it 'creates new booking' do
+      post :create, params: { _json: [
+                                        {name: 'id:name', value: 'Test'},
+                                        {name: 'id:email', value: 'example@abc.com'},
+                                        {name: 'category', value: 'electronics'},
+                                        {name: 'items', value: 'camera'},
+                                        {name: 'ItemQuantity', value: '10'},
+                                        {name: 'PurposeofRental', value: "Test\r\nMultiline"}
+                                     ],
+                              booking: {} }
```

Comment:
Also sanity check: this is added because the way the `curl` post request is done gives this random `booking: {}` as one of the params.